components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -5294,11 +5256,34 @@ public void addURLTemplates(int apiId, API api, Connection connection) throws AP
         }
     }
 
+    /**
+     * Fetches an Application with OAuth Apps, by name.
+     *
+     * @param applicationName Name of the Application
+     * @param userId          Name of the User.
+     * @param groupId         Group ID
+     * @throws APIManagementException
+     */
+    public Application getApplicationWithOAuthApps(String applicationName, String userId, String groupId)
+            throws APIManagementException {
+
+        Application application = getApplicationByName(applicationName, userId, groupId);
+
```

Comment:
Additional white spaces

Reply:
are you referring to the new line? if yes, that was added for clarity. but can remove them if need. wdyt?

Reply:
New lines before the for loop and return statement looks clean.
Additional new lines between method variable initialization looks bit noisy to me :)

Reply:
Got it. Fixed it.